app/models/manageiq/providers/redhat/infra_manager/refresh/strategies/api3.rb

```diff
@@ -1,5 +1,9 @@
 module ManageIQ::Providers::Redhat::InfraManager::Refresh::Strategies
   class Api3 < ManageIQ::Providers::Redhat::InfraManager::Refresh::Refresher
+    def self.ems_type
+      @ems_type ||= "rhevm".freeze
```

Comment:
It is needed due to our refresh structure using strategies. We call `parent.ems_type` which in our case is `ManageIQ::Providers::Redhat::InfraManager::Refresh::Strategies` which do not have required method.